database/migration/aruha-522-timelines.sql

```diff
@@ -0,0 +1,23 @@
+SET ROLE zalando_nakadi_data_owner;
+
+CREATE TABLE zn_data.storage (
+  st_id            VARCHAR(36) NOT NULL PRIMARY KEY,
+  st_type          VARCHAR(32) NOT NULL,
+  st_configuration JSONB       NOT NULL
+);
+
+
```

Comment:
Yes, you will never have matching column names across tables. Since the old rules are not forced anymore, you can adopt own ones as a team.

I just noted some inconsistency in your naming schema and wanted to make sure you know about the former conventions.